app/scripts/modules/kubernetes/serverGroup/configure/wizard/Clone.controller.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.serverGroup.configure.kubernetes.clone', [
+  require('angular-ui-router'),
+  require('../../../../core/application/modal/platformHealthOverride.directive.js'),
```

Comment:
You should include `require` statements for the dependencies of the controller - `serverGroupWriter`, `modalWizardService`, `taskMonitorService`, `kubernetesServerGroupConfigurationService`. Then, if you write tests for this controller, you won't have to `require` them into the tests.